design/help.md

```diff
@@ -0,0 +1,84 @@
+# Design of the Help plugin
+
+This document describes the design of the Help plugin. This document illustrates how
+our research on personas and other solutions translates into the design decisions made to improve the Help plugin.
+
+# Personas
+
+## Tim Burns
+
+Tim Burns works at Bloomberg as a marketing data scientist. He takes 
```

Comment:
Let's not use a specific company name. Instead, you can use a generic name like `Acme Widgets` or `SomeCompany`.